### PR TITLE
docs(react-theme): Update readme

### PR DESCRIPTION
**Pull request by miroslavstastny**
Update readme in `react-theme`:
- delete "not production-ready" warning
- add usage
- add contributing guide

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |   `78.914 kB`<br />`24.06 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `83.228 kB`<br />`20.889 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `48.381 kB`<br />`13.696 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |    `14.95 kB`<br />`5.989 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `68.349 kB`<br />`19.039 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |     `22.6 kB`<br />`7.205 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `23.503 kB`<br />`7.497 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |    `24.05 kB`<br />`7.067 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |    `35.836 kB`<br />`9.59 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `42.862 kB`<br />`10.808 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `38.454 kB`<br />`10.461 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `45.87 kB`<br />`11.811 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `51.017 kB`<br />`11.007 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `67.002 kB`<br />`19.261 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `62.684 kB`<br />`18.177 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `8.561 kB`<br />`3.601 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |     `9.604 kB`<br />`3.94 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `8.662 kB`<br />`3.656 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `73.863 kB`<br />`24.018 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |   `73.45 kB`<br />`23.927 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `187.656 kB`<br />`51.96 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `33.359 kB`<br />`11.004 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `80.608 kB`<br />`24.053 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `16.459 kB`<br />`5.902 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |    `10.78 kB`<br />`4.264 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `23.598 kB`<br />`7.661 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |     `9.338 kB`<br />`3.86 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `12.254 kB`<br />`4.956 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `115.735 kB`<br />`35.419 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `118.934 kB`<br />`35.916 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `10.685 kB`<br />`4.104 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `102.938 kB`<br />`31.542 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |   `10.576 kB`<br />`3.875 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `5.851 kB`<br />`1.964 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |     `19.7 kB`<br />`7.404 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `15.755 kB`<br />`5.883 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |   `35.56 kB`<br />`11.929 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |     `14.248 kB`<br />`5.7 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `20.846 kB`<br />`7.346 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `31.526 kB`<br />`10.046 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `43.943 kB`<br />`12.382 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `19.977 kB`<br />`6.438 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `32.097 kB`<br />`10.27 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `11.782 kB`<br />`4.605 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `15.092 kB`<br />`5.044 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `23.988 kB`<br />`8.011 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |   `29.548 kB`<br />`6.434 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |   `17.452 kB`<br />`5.054 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `41.502 kB`<br />`14.623 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8ebc00ac214d6d86c6666b0b7fa0489076c42dc7'>8ebc00ac214d6d86c6666b0b7fa0489076c42dc7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/Avatar-mount_base.html">1277</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/Avatar-mount.html">1258</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/Button-mount_base.html">926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/Button-mount.html">933</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProvider-mount_base.html">1508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProvider-mount.html">1481</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProviderWithTheme-mount_base.html">574</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProviderWithTheme-mount.html">566</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">538</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">579</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/MakeStyles-mount_base.html">1969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/MakeStyles-mount.html">1936</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/SpinButton-mount_base.html">2318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24755/perf-test-react-components/SpinButton-mount.html">2306</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by bsunderhus — APPROVED** [review]

File: change/@fluentui-react-theme-2a665c75-048e-48df-a230-ff78408c7129.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: Update readme",
```
Comment (layershifter) [inline change/@fluentui-react-theme-2a665c75-048e-48df-a230-ff78408c7129.json:3]:
```suggestion
  "comment": "docs: Update readme",
```

File: packages/react-components/react-theme/README.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+};
+
+const customTheme = createWebLightTheme(customBrandRamp);
+```
+
+## Contributing
+
+### Updating colors in the theme
+
+The color tokens are generated by token pipeline. Do not edit colors directly. Instead, update the token pipeline first and once the changes are merged there, run:
```
Comment (layershifter) [inline packages/react-components/react-theme/README.md:51]: > Instead, update the token pipeline first and once the changes are merged there

Hmm... Where? 🐱

Comment (miroslavstastny) [reply packages/react-components/react-theme/README.md:51]: currently private

**Pull request merged**